mod_reforged/hooks/items/ammo/ammo.nut

```diff
@@ -0,0 +1,76 @@
+::mods_hookExactClass("items/ammo/ammo", function(o)
+{
+    o.m.AmmoTypeName <- "ammo_type_name";
+    o.m.AmmoWeight <- 0.0;
+    o.m.AddGenericSkill <- true;
```

Comment:
When would this be false?

Reply:
I don't know. I copied the way Vanilla did it in other places. In those places this variable is false by default, probably so that those items don't polute the skill-container when not necessary.

I'll remove it